frontend/tests/page-object/dashboard.page.ts

```diff
@@ -0,0 +1,68 @@
+import type { Page } from "@playwright/test";
+import { expect } from "@playwright/test";
+
+export const DashboardPage = {
+    overview: (page: Page) => page.locator(".overview"),
+    onlineValidators: (page: Page) => page.locator("text=Online Validators"),
+    heatmap: (page: Page) => page.locator(".heatmap"),
+    manageValidatorsButton: (page: Page) => page.locator("text=Manage Validators"),
+    validatorsButton: (page: Page) => page.getByRole("button", { name: "Validators" }),
+    accountsButton: (page: Page) => page.getByRole("button", { name: "Accounts Coming soon" }),
+    continueButton: (page: Page) => page.getByRole("button", { name: "Continue" }),
+    addDashboardTitle: (page: Page) => page.getByText("Add a new dashboard"),
+    ethereumOption: (page: Page) => page.getByRole("button", { name: "Ethereum" }),
+    gnosisOption: (page: Page) => page.getByRole("button", { name: "Gnosis" }),
+    backButton: (page: Page) => page.locator("text=Back"),
+    continueNetworkButton: (page: Page) => page.getByRole("button", { name: "Continue" }),
+
+    clickContinue: async (page: Page) => {
```

Comment:
would be nice if we use the most generic form of our `app behaviour`, which does not rely on the `users device`.
Do I really `click` on a `mobile device`?

What if I tell `alexa` or `siri` that I want to do something specific....if tests could be reduced to the actual `behavioural pattern` we will have the most `resilient` tests 

```suggestion
    continue: async (page: Page) => {
```